src/table.rs

```diff
@@ -75,6 +75,7 @@ impl Table {
         let mut s = Vec::new();
 
         for change in &mut self.changes {
+            println!("in make's change loop change: {:?}", change.clone());
```

Comment:
It kinda feels like this made it in here by accident? :)

Reply:
hehe, yes.